### ISSUE TITLE
Impossible sending muliple persons. 

### ISSUE THREAD

**Issue by Didier83** [SMSSecure/SMSSecure#239]
When I choose a contact using the pencil in the right down coin, the application display directly the redaction part to write message, but I have no way to add a new receiver. When I click on the receiver, I have information about him/her, but always no way to add a nex receiver...! I have a Sony Xperia z3 under Lollipop.
Thanks for your help..!!

**Comment by MicroJoe** [comment]
You need to create a group before starting a conversation with multiple persons using MMS.

**Comment by BastienLQ** [comment]
Duplicate of #130.
